issuetracker/pivotaltracker/pivotaltracker_test.go

```diff
@@ -0,0 +1,34 @@
+package pivotaltracker
```

Comment:
could you also make the changes for the `redmine` issue tracker?

Reply:
Use regex here?

```
// IssueAPIOrigin returns the URL for redmine's issue-fetching API
func (it *IssueTracker) issueAPIOrigin() string {
	return fmt.Sprintf("%s/issues/", it.Origin)
}
```

Reply:
Oh, you're right there is no point to make these changes there. It is simple enough